src/Rule/RuleList.php

```diff
@@ -22,7 +22,7 @@ final class RuleList implements Stringable
      */
     public function __construct(
         public readonly Filter|null $globalFilter = null,
-        public readonly array $rules,
+        public readonly array $rules = [],
```

Comment:
An empty array as a default doesn't make sense here, the order should be fixed instead and rules should become the first param. ;)

Reply:
I'll need this change in #11 so I'll fix it there!